processor/src/test/java/org/mapstruct/ap/testutil/runner/CompilingStatement.java

```diff
@@ -102,12 +102,7 @@ protected CompilationCache initialValue() {
         }
     };
 
-    private static final List<String> LIBRARIES = Arrays.asList(
```

Comment:
Hmm, works for me. In what way does the test fail for you? Did you mvn clean install first?